cash_register.rb

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env ruby
+
+# This is the CashRegister Class
+class CashRegister
+  def initialize
+    @total = 0.00
+    two_decimal = format('%.2f', @total)
+    puts "The cash register balance is $#{two_decimal}"
+  end
+
+  def purchase(item)
+    food_items = ['Organic Bears Milk', 'Wagyu Ribeye', 'Smoked Bacon Salt']
```

Comment:
Nothing in the requirements talked about `food_items`...  Remove this code please.